src/jira/client/index.ts

```diff
@@ -243,7 +243,7 @@ export default async (
 	gitHubInstallationId: number,
 	logger?: Logger
 ) => {
-	return await getJiraClient(jiraHost, gitHubInstallationId, logger);
```

Comment:
I'm not sure I like the linting rule that makes us remove "redundant" `await`s . If someone adds some logic to this code after line 246, they might forget adding the `await` back in ...

Reply:
yeah, but then the linter would complain that there's a missing await :)

Plus, if you add code after this, you'll have to change the return altogether.